tasks/removeTokens.ts

```diff
@@ -1,50 +1,27 @@
-import { ContractTransaction } from 'ethers';
```

Comment:
What is the expectation of the removal?

I did some testing, and it fails in some scenarios, but I'm not sure if are covered. 

For ie:

- Remove tokens that are not allowed
- Remove tokens that are in a specific verifier without specifying it

Reply:
resolved in a discussion with @antomor